lib/invoca/utils.rb

```diff
@@ -20,9 +20,7 @@ module Utils
 require "invoca/utils/guaranteed_utf8_string"
 require "invoca/utils/version"
 
-unless defined?(Diff)
-  Diff = Invoca::Utils::Diff
-end
+Diff = Invoca::Utils::Diff
```

Comment:
This has the side-effect of adding a "previously initialized constant" warning to the logs whenever invoca-utils is loaded. See the demo for an example. Is that amount of noise a deal breaker?